e2e/container/create_test.go

```diff
@@ -6,10 +6,16 @@ import (
 
 	"github.com/docker/cli/e2e/internal/fixtures"
 	"github.com/docker/cli/internal/test/environment"
+	"gotest.tools/fs"
 	"gotest.tools/icmd"
 	"gotest.tools/skip"
 )
 
+const (
+	evilNotaryURL = "https://evil-notary-server:4444"
+	evilImageName = "registry:5000/evil-alpine:latest"
```

Comment:
Should these be put in e2e/internal/fixtures/fixtures.go?

Reply:
Yes, you are right, this should be in `fixtures.go`. I am going to move them.